gadget/params.c

```diff
@@ -312,6 +312,21 @@ create_gadget_parameter_set()
     param_declare_double(ps, "NuPartTime", OPTIONAL, 0.3333333, "Scale factor at which to turn on hybrid neutrino particles.");
     /*End parameters for the massive neutrino model*/
 
+    /*Parameters for the Excursion Set Algorithm*/
+    param_declare_int(ps, "ExcursionSetFlag", OPTIONAL, 0, "Use the excursion set instead of the global UV field");
+    param_declare_int(ps, "UVBGdim", OPTIONAL, 64, "Side length of the excursion set grid.");
+    param_declare_int(ps, "UVBGSaveGrids", OPTIONAL, 1, "Flag to save reionisation grids.");
+    param_declare_int(ps, "ReionFilterType", OPTIONAL, 0., "Filter type for Excursion set");
```

Comment:
What are the possibilities for this option?

Reply:
Filter type for Excursion set: 0 = real-space top-hat, 1 = k-space top-hat, 2 = gaussian. I have gone back and updated the description for this parameter now and will push it with the other changes

Reply:
Ah I see now, thanks!